docs/Configuration.md

```diff
@@ -289,6 +292,34 @@ Each locale set in the configuration file, including the implied
 ".UTF-8", must also be installed or activate on the system
 running the RPCAPI daemon for the translation to work correctly.
 
+## METRICS section
+
+### statsd_host
+
+An [LDH domain name] or IP address.
+
+The host name of the machine on which the StatsD receiver is running.
+
+Leave unspecified to disable the metrics.
+
+Note that this feature requires the `Net::Statsd` module to be installed.
+
+### statsd_port
+
+The port the StatsD receiver is listening on.
+Default value: `8125`.
+
+### Available metrics
```

Comment:
I do not think that the section "Available metrics" and its text belong to the `docs/Configuration.md` file. It probably belong to a new document for metrics where it could also be shortly described how the function is used.

Reply:
I moved it to a `docs/Telemetry.md` document